thrusters_8:
  ros__parameters:
    serial_port: "/dev/ttyUSB0"
    topic_name: "cmd_vel"